### PR TITLE
feature vip page

### PR DESCRIPTION
**Pull request by smailzhu**

### CODE REVIEW

File: src/components/card.js [hunk 1]
```diff
@@ -234,7 +234,76 @@ export default class Card {
...
+                let userDetails = JSON.parse(localStorage.getItem('loggedInUser'));
+                let balance = window.Database.getBalance(userDetails['username']);
+                if (amount > balance){
+                    alert("balance not enough");
+                    return ;
+                }
+                balance -= amount;
+                console.log("balance 2: "+balance);
+                window.Database.changeBalance(userDetails['username'], balance);
+                let vipId = "vip" + userDetails['userID'];
```
Comment (ahenningsson) [inline src/components/card.js:269]: In a more practical example the table-number would probably not have anything to do with who is sitting at the table, but rather that each tablet on each table has a table-value set, which would be used here.

File: src/components/card.js [hunk 2]
```diff
@@ -234,7 +234,76 @@ export default class Card {
...
+                    amount += parseFloat(response.pris);
+                }
+                let userDetails = JSON.parse(localStorage.getItem('loggedInUser'));
+                let balance = window.Database.getBalance(userDetails['username']);
+                if (amount > balance){
+                    alert("balance not enough");
+                    return ;
+                }
+                balance -= amount;
+                console.log("balance 2: "+balance);
```
Comment (ahenningsson) [inline src/components/card.js]: Remove "console.log()"

File: src/components/card.js [hunk 3]
```diff
@@ -234,7 +234,76 @@ export default class Card {
          * Checks if the user is logged in as staff
          */
         if(this.isVip){
+            let order_num = $('<select name="order-num" class="order-num order-num-'+response.id+'">' +
+                '<option value="0">cups ?</option>' +
```
Comment (ahenningsson) [inline src/components/card.js]: "cups ?" should be decide by a <span>-tag which uses i18n. Should say something like "Amount:"/"Antal:"

**Review by ahenningsson — COMMENTED** [review]
👍

**Review by ahenningsson — COMMENTED** [review]
👍

File: src/components/card.js (hunk 2 above)
Comment (smailzhu) [reply src/components/card.js]: 2f12e1c Done

File: src/components/card.js (hunk 3 above)
Comment (smailzhu) [reply src/components/card.js]: It make sense. Fixed in fa1b10f

**Pull request merged**